data/ecosystem/vendors.yaml

```diff
@@ -321,3 +321,10 @@
   contact: 'info@greptime.com'
   oss: true
   commercial: true
+name: ClickHouse
+  distribution: false
+  nativeOTLP: true
+  url: 'https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/clickhouseexporter'
```

Comment:
That doesn't fit, if this would be a support for native OTLP you would not need a vendor specific exporter

Reply:
I must have misinterpreted what "native OTLP" means. If it means that we support OTLP protocol in ClickHouse database itself -- then yes, it is not the case, we do require an exporter. Then, we should change it to false.

Reply:
then set it to false please

Reply:
Yes, just wanted to triple check my understanding. I set "nativeOTLP: false".